Libraries/LibWeb/Layout/TreeBuilder.cpp

```diff
@@ -843,6 +850,27 @@ void TreeBuilder::remove_irrelevant_boxes(NodeWithStyle& root)
         box->parent()->remove_child(*box);
 }
 
+// Fixup pseudo-elements with content containing reversed counters.
+// Why? The final value for an uninitialized reversed counter cannot be calculated
+// in one pass, since it depends on the number of counter-increment occurrences.
+// Before this call, values of reversed counters will be incorrect (exception: values after a counter-set).
+// We recalculate the correct value for with all those occurrences.
+void TreeBuilder::fixup_reversed_counters_content()
+{
+    VERIFY(!m_content_with_reversed_counters_fixup_queue->is_empty());
+
+    for (auto const& item : *m_content_with_reversed_counters_fixup_queue) {
+        auto pseudo_element_style = item.element->pseudo_element_computed_properties(item.pseudo_element); // Will eventually resolve all reversed counters used in content.
+        DOM::AbstractElement element_reference { *item.element, item.pseudo_element };
```

Comment:
If `ElementWithReversedCounterContent` holds an `AbstractElement`, we can ask that for the computed properties and it'll work with regular elements once we need that.